src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -24,6 +24,9 @@
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" Version="15.0.26730-alpha" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Framework" Version="15.0.26730-alpha" />
+    <PackageReference Include="Microsoft.VisualStudio.TextManager.Interop" Version="7.10.6070" />
```

Comment:
@jasonmalinowski Is it ok to reference these here from this package? Will this be a concern at all for vs4mac?

Reply:
Correct, that can't go there. But this is PR for Personal Review, so at this point @JieCarolHu is still getting things working.

Reply:
removed